modules/ROOT/pages/configure-message-flows.adoc

```diff
@@ -3,24 +3,56 @@ ifndef::env-site,env-github[]
 include::_attributes.adoc[]
 endif::[]
 
-Message flow diagram here? - Inbound transmission - validate transmission payload - route to message flows - route through message flows
+To receive messages from a partner, create a message flow and configure its sections for both the source and target. For the Endpoint, source at host - this is the origin of the message flow when defined by a host endpoint and message type, often an API. For the endpoint, target at host - this is the final destination of a message flow when defined by a host endpoint and message type, often a Host API.  
 
-When receiving a B2B message from your trading partner, there may be more than one URL.
-For example, a receiving URL, and a URL for receiving asynchronous MDNs (Message Disposition Notifications).
-If this is the case, you only need to configure the Receiving URL.
+For example, a distributor might group a set of purchase orders by sending them in one transmission to a supplier trading partner.
+
+You'll need the following for the Source:
```

Comment:
To configure the source-at-host endpoint, you must have the following information:
 (This is a wording guess, because I'm not sure about these "sections.")

Reply:
Gonna go to the next edit.